src/routes/+page.svelte

```diff
@@ -1,2 +1,40 @@
-<h1>Welcome to SvelteKit</h1>
-<p>Visit <a href="https://kit.svelte.dev">kit.svelte.dev</a> to read the documentation</p>
+<h1>Welcome to Kendo Club!</h1>
+
+<h2>What is Kendo?</h2>
+<p>
+	Kendo is a Japanese martial art which literally translates to “The Way of the Sword”. It's an
+	athletic sport which is played by means of one-on-one striking between opponents using shinai
+	(sword made from bamboo slats) while wearing kendo-gu (protective armor).
+</p>
+
+<h2>Who are we?</h2>
+
+Kendo Club at the University of Michigan is led by students and instructed by highly experienced
```

Comment:
This text should be inside a block level element such as `<p>`

Reply:
Fixed in [b080c7b](https://github.com/kendoclub-at-umich/website/pull/31/commits/b080c7b3d7c7ad5eddbc897b8a30bd36f27b3512)